module/platform/webots/Webots/src/Webots.cpp

```diff
@@ -214,11 +231,10 @@ namespace module::platform::webots {
         error_io.unbind();
         shutdown_handle.unbind();
 
-        int fd = tcpip_connect(server_address, port);
-
+        const int fd = tcpip_connect(server_address, port);
 
         char initial_message[8];
-        int n = recv(fd, initial_message, sizeof(initial_message), 0);
+        const int n = recv(fd, initial_message, sizeof(initial_message), 0);
 
         if (n > 0) {
             if (strncmp(initial_message, "Welcome", sizeof(initial_message)) == 0) {
```

Comment:
We should probably just convert initial message to a proper string above and just use `==` to compare. Did this [here in Mikyla's PR](https://github.com/NUbots/NUbots/pull/422/commits/6d40546fad6da098a386e9bc2a9a4df133e735d1#diff-fc0b2b2d580a7b5d606168708dc45d1b335285a24d5c80f1f4ef7846672cc8b4L176-R223) but it got overwritten in the merge.